score/etl/score/etl.py

```diff
@@ -0,0 +1,378 @@
+import collections
+import functools
+import pandas as pd
+
+from etl.base import ExtractTransformLoad
+from utils import get_module_logger
+from etl.sources.census.etl_utils import get_state_fips_codes
+
+logger = get_module_logger(__name__)
+
+
+class ScoreETL(ExtractTransformLoad):
+    def __init__(self):
+        # Define some global parameters
+        self.GEOID_FIELD_NAME = "GEOID10"
```

Comment:
Could we maybe move `GEOID_FIELD_NAME` and `GEOID_TRACT_FIELD_NAME` to constants set across multiple classes for consistency -- e.g., on the base `ExtractTransformLoad` class, or in some shared constants file?

Reply:
Good idea. Done.